jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/common/Progress.java

```diff
@@ -14,9 +13,13 @@ public String getColor() {
     @JDIAction(value = "Get '{name}' width")
     public String getWidth() {
         if (uiElement.hasAttribute("style")) {
-            return uiElement.core().attr("style");
+            if (uiElement.core().attr("style").contains("width")) {
+                return uiElement.core().attr("style");
+            } else {
+                return "width: 0%; or element is wrong";
+            }
         } else {
-            return "width: 0%;";
+            return "width: 0%; or element is wrong";
```

Comment:
it's bad way to return custom value on error. we should to send null or empty (for example, I suggest you send the same as selenium getText() do it when there is no text or getValue() when there is no value)